### DISCUSSION TITLE
Things that would improve leather engine and thus become better than psych engine

### DISCUSSION THREAD

**Discussion by BlueDasher334** [Leather128/LeathersFunkinEngine#227] [Ideas]
1.- Local Multiplayer system ( pls i need before 14/05/2022 

2.- An option to test the chart from the debug mode

3.- to be able to change the debug key ( 7,8 ) since sometimes when you are playing you usually press it unintentionally

4.- a more optimised debug as in psych engine (to avoid lag)

5.- (optional) an option to change the pitch of the song without changing the speed 

6.- an option for the opponent or player 1 to have a different ui skin

7.- compatibility with all modcharts

8.- creator of personalized notes

9.- new events such as: change the number of notes

10.- a system so that the engine is no longer restarted and thus not lose data

11.- when you are putting the icons in optimizations, that they appear even if they are not the pardons

12.- creator of weeks

13.- an improved freeplay (such as FNF modding plus) through an option 

14.- a note of alt animations

15.- the events of the shaggy xd mod

16.- a list in the files for the particular color of the notes

**Comment by Leather128** [comment]
yeah sorry but i'm not doing all that, especially not for the inferior haxeflixel version of leather engine
